app/controllers/api/v1/reactions_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class ReactionsController < ApiController
+      before_action :authenticate!
+
+      def create
+        remove_count_cache_key
+
+        result = ReactionToggle.toggle(params, current_user: current_user || @user)
+
+        if result.success?
+          render json: { result: result.action, category: result.category }
```

Comment:
Will we amend the respond body to contain some additional json like outlined in the [issue](https://github.com/forem/forem/issues/18263)?

```
Return body:
id: integer
reactible_type: enum[article, comment, user]
reactible_id: integer
category: enum[like, readinglist, unicorn, thinking, hands, thumbsup, thumbsdown, vomit]
```

Reply:
Good catch! :+1: